assignments/objects.js

```diff
@@ -74,29 +74,29 @@ console.log(personFive.gender)
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
-const personTwo = {
-  id : 2,
-  name : "Kennan",
-  email : "kdiben1@tinypic.com",
-  gender: "M",
-  sayHello : function(){
+// const personTwo = {
+//   id : 2,
+//   name : "Kennan",
+//   email : "kdiben1@tinypic.com",
+//   gender: "M",
+  Kennan.sayHello = function(){
```

Comment:
Concept is correct, but still getting an error because the variable `Kennan` is not defined. There are two ways to approach this.
```
personTwo.sayHello = function(){
    console.log("Hello, my name is" + ' ' + this.name + "!")
  }
```
or 
```
const personTwo = {
  id : 2,
  name : "Kennan",
  email : "kdiben1@tinypic.com",
  gender: "M",
  sayHello: function() {
    return `Hello, my name is ${this.name}`
  }
}

console.log(personTwo.sayHello())
```